pkg/resources/standard_database_acceptance_test.go

```diff
@@ -0,0 +1,562 @@
+				ConfigVariables:         configVariables(id, comment, 20, 30, true, "en_US", sdk.StorageSerializationPolicyCompatible, sdk.LogLevelInfo, sdk.TraceLevelOnEvent, true, true),
+				ResourceName:            "snowflake_standard_database.test",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"replication.0.ignore_edition_check"},
+			},
+		},
+	})
+}
+
+func TestAcc_StandardDatabase_Update(t *testing.T) {
+	id := acc.TestClient().Ids.RandomAccountObjectIdentifier()
+	secondaryAccountIdentifier := acc.SecondaryTestClient().Account.GetAccountIdentifier(t).FullyQualifiedName()
+	comment := random.Comment()
+
+	externalVolumeId, externalVolumeCleanup := acc.TestClient().ExternalVolume.Create(t)
+	t.Cleanup(externalVolumeCleanup)
+
+	catalogId, catalogCleanup := acc.TestClient().CatalogIntegration.Create(t)
+	t.Cleanup(catalogCleanup)
+
+	configVariables := func(
+		id sdk.AccountObjectIdentifier,
+		comment string,
+		dataRetention int,
+		maxDataExtension int,
+		replaceInvalidCharacters bool,
+		defaultDdlCollation string,
+		storageSerializationPolicy sdk.StorageSerializationPolicy,
+		logLevel sdk.LogLevel,
+		traceLevel sdk.TraceLevel,
+		withFailover bool,
+		ignoreEditionCheck bool,
+	) config.Variables {
+		return config.Variables{
+			"name":                            config.StringVariable(id.Name()),
+			"comment":                         config.StringVariable(comment),
+			"transient":                       config.BoolVariable(false),
+			"data_retention_time_in_days":     config.IntegerVariable(dataRetention),
+			"max_data_extension_time_in_days": config.IntegerVariable(maxDataExtension),
+			"external_volume":                 config.StringVariable(externalVolumeId.Name()),
+			"catalog":                         config.StringVariable(catalogId.Name()),
+			"replace_invalid_characters":      config.BoolVariable(replaceInvalidCharacters),
+			"default_ddl_collation":           config.StringVariable(defaultDdlCollation),
+			"storage_serialization_policy":    config.StringVariable(string(storageSerializationPolicy)),
+			"log_level":                       config.StringVariable(string(logLevel)),
+			"trace_level":                     config.StringVariable(string(traceLevel)),
+			"account_identifier":              config.StringVariable(secondaryAccountIdentifier),
+			"with_failover":                   config.BoolVariable(withFailover),
+			"ignore_edition_check":            config.BoolVariable(ignoreEditionCheck),
+		}
+	}
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
+		PreCheck:                 func() { acc.TestAccPreCheck(t) },
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.RequireAbove(tfversion.Version1_5_0),
+		},
+		CheckDestroy: acc.CheckDestroy(t, resources.StandardDatabase),
+		Steps: []resource.TestStep{
```

Comment:
It doesn't test update because we're providing just one config, am I missing something?

Reply:
True, I think this test is obsolete because every update path is tested in the other tests, but I can update this one to contain all of them (I'll do it tomorrow).

Reply:
Done